### ISSUE TITLE
Support more API query parameters in find_groups()

### ISSUE THREAD

**Issue by missaugustina** [rladies/meetupr#21]
To find R-Ladies meetups that aren't using the topic, I do a text search and limit to a category and then apply a regex to the result. We should support category as a parameter.

 (TODO: make a reprex)

```  
groups_url <- "https://api.meetup.com/find/groups"

  groups_query_params <- list(
    key=params$meetup_api_key, 
    sign=TRUE,
    page=200,
    radius="global")
  
  # by topic
  meetup_groups_topic <- get_meetups(groups_url, append(groups_query_params, c(topic_id=1513883, order="members")))
  
  # by text + category
  meetup_groups_text <- get_meetups(groups_url, append(groups_query_params, c(text="r-ladies", category=34)))
  
  meetup_groups_aggr <- bind_rows(meetup_groups_topic, meetup_groups_text %>% anti_join(meetup_groups_topic, by="id"))
  meetup_groups <- meetup_groups_aggr %>% filter(str_detect(name, "[Rr]([ -]?)[Ll]adies"))`
```

Design question - do we want to explicitly support parameters or do we want to consider dynamic arguments?

**Comment by ledell** [comment]
So far, we've explicitly added the arguments, but if it makes sense to group new ones together, we could do that.  Lists are nice because you never really have to change the function API (new args just go inside the list) however it's much harder to document the options when they are named list elements list instead of explicit args.

**Comment by missaugustina** [comment]
Yeah I'm on the fence myself. I like having explicit support and documentation but then we're really just supposed to be an interface for them. I'll stick with the current paradigm for now.

**Comment by maelle** [comment]
More generally I suppose one could support all documented parameters https://www.meetup.com/fr-FR/meetup_api/docs/find/groups/?uri=%2Fmeetup_api%2Fdocs%2Ffind%2Fgroups%2F
